src/Features/authentication.feature

```diff
@@ -35,43 +34,79 @@ Feature: When an user needs to authenticate
-      | info   | /Setting SARI '.*' for identifier '.*'/ | present |
-      | info   | Return authentication page with QR code                                                                                                 | present |
-      | info | Using dummy as UserStorage encryption type | present |
-      | info | Client request QR image | present |
-      | info | Return QR image response | present |
-      | notice | Got POST with login response | present |
-      | notice | Received login action from client with User-Agent "Symfony" and version "" | present |
-      | info | Validating authentication response | present |
-      | notice | /Authenticated user ".*" in session ".*"/ | present |
-      | info | response is valid | present |
-      | notice | User authenticated OK | present |
-
-      | info   | Using dummy as UserStorage encryption type                                                                                              | present |
-      | info | Verifying if there is a pending authentication request from SP | present |
-      | info | Verify if user is blocked | present |
-      | info   | Verifying if authentication is finalized                                                                                                | present |
-      | info   | Authentication is finalized, returning to SP                                                                                            | present |
-      | notice | Application authenticates the user                                                                                                      | present |
-      | notice | Created redirect response for sso return endpoint "/saml/sso_return"                                                                    | present |
-      | notice | Received sso return request                                                                                                             | present |
-      | info   | Create sso response                                                                                                                     | present |
-      | notice | /Saml response created with id ".*", request ID: ".*"/                                                                                  | present |
-      | notice | Invalidate current state and redirect user to service provider assertion consumer url "https://tiqr.dev.openconect.local/demo/sp/acs"     | present |
-      | info   | /SAMLResponse with id ".*" was not signed at root level, not attempting to verify the signature of the reponse itself/                  |         |
-      | info   | /Verifying signature of Assertion with id ".*"/                                                                                         |         |
+      | info   | AuthnRequest stored in state                                                                                                              |         |
```

Comment:
It seems this is done by the Stepup GSSP bundle. 
So we are testing the behaviour of an external dep here.

![image](https://github.com/user-attachments/assets/252dc0a0-fd39-4a3e-9097-48cce0e9e7c4)

Reply:
Could this be because of the log level that info messages are not logged.

We want the SARI (SAML Authentication Request ID) to be logged at least once for each HTTP requests to a GSSP. This could also be as part of a sari in the logger conext.

Reply:
Discussed: Not caused by these changes. Will be noticed and fixed it its actually a problem (probably not)